docs/content/manual/backup-and-restore/4954-cannot-resotre-new-vm-have-snapshot.md

```diff
@@ -0,0 +1,27 @@
+---
+title: VM have snapshot can't be restored to new VM
+---
+
+* Related issues: [#4954](https://github.com/harvester/harvester/issues/4954) [BUG] VM Restore to new VM doesn't work if there is VM Snapshot
+
+
+## Category: 
+* Backup and Restore
+
+## Verification Steps
+1. Prepare a 3 nodes Harvester cluster
+1. Create a vm named vm1
+1. Setup nfs backup target
+1. Create a backup for the vm1
+1. Create a snapshot for the vm1
+1. Restore the backup of vm1 to create a new VM
+1. Check can restore vm correctly
+1. Shutdown vm1
+1. Restore the backup of vm to replace the existing vm
+1. Select Retain volume
+1. Check can restore vm correctly
```

Comment:
Could you check if this is covered in automated test already?

Reply:
By checking our existing vm backup and snapshot integration test scenarios in:
1. test_4_vm_backup_restore.py
2. test_4_vm_snapshot.py

I did not found a suitable automation test could both cover when vm already have the snapshot and backup on it. 

Among these tests

#### test_4_vm_backup_restore.py

1. test_connection
2. tests_backup_vm
3. test_update_backup_by_yaml
4. test_restore_with_new_vm
5. test_restore_replace_with_delete_vols
6. test_restore_replace_vm_not_stop
7. test_restore_with_invalid_name
8. test_backup_migrated_vmtest_restore_replace_migrated_vm
9. test_backup_multiple
10. test_delete_last_backup
11. test_delete_middle_backup

####  test_4_vm_snapshot.py 
1. test_vm_snapshot_create
2. test_restore_into_new_vm_from_vm_snapshot
3. test_replace_is_rejected_when_deletepolicy_is_retain
4. test_replace_vm_with_vm_snapshot
5. test_restore_from_vm_snapshot_while_pvc_detached_from_source
6. test_create_vm_snapshot_while_pvc_detached
7. test_vm_snapshots_are_cleaned_up_after_source_vm_deleted
12. test_volume_snapshots_are_cleaned_up_after_source_volume_deleted